### PR TITLE
add option to run without integration tests

### PR DESCRIPTION
**Pull request by christian-posta**
and clean up read me a little

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,19 +74,34 @@ Using Docker has several advantages:
...
     $ mvn clean install
 
-The build starts and uses several Docker containers for different DBMSes. Note that if Docker is not running, you'll likely get an arcane error -- if this is the case, always verify that Docker is running, perhaps by using `docker ps` to list the running containers.
+The build starts and uses several Docker containers for different DBMSes. Note that if Docker is not running or configured, you'll likely get an arcane error -- if this is the case, always verify that Docker is running, perhaps by using `docker ps` to list the running containers.
+
+### Don't have docker running locally for builds?
+
+You can skip the integration tests and docker-builds with the following command:
 
+    $ mvn clean install -DskipIT -P -docker
```
Comment (rhauch) [inline README.md]: How about using a single property to turn off integration tests? 

    mvn clean install -DskipIT

Could be done by making the `docker` profile run except when the `skipIT` is set? WDYT?

Comment (christian-posta) [inline README.md]: Yah! Sounds good to me! Will update it in a bit.
Thanks!

On Thu, Feb 11, 2016 at 4:04 PM, Randall Hauch <notifications@github.com>
wrote:

> In README.md
> <https://github.com/debezium/debezium/pull/12#discussion_r52682708>:
>
> >
> > +    $ mvn clean install -DskipIT -P -docker
>
> How about using a single property to turn off integration tests?
>
> mvn clean install -DskipIT
>
> Could be done by making the docker profile run except when the skipIT is
> set? WDYT?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/debezium/debezium/pull/12/files#r52682708>.
>



-- 
*Christian Posta*
twitter: @christianposta
http://www.christianposta.com/blog
http://fabric8.io

**Comment by rhauch** [comment]
@christian-posta, can you rebase on the latest `master` and update this PR? Thanks!

**Comment by christian-posta** [comment]
@rhauch so i've rebased and fixed conflicts. one thing... are you expecting the alt-mysql profile (when enabled) to build/test successfully?

**Comment by rhauch** [comment]
@christian-posta: no, the `alt-mysql` profile is only there for manually running and debugging with the Docker-created MySQL image (rather than the MySQL Team created image we use in our normal builds).   See https://github.com/debezium/debezium/blob/master/debezium-connector-mysql/NOTES.md for more detail.

**Comment by rhauch** [comment]
@christian-posta: something is wrong with the `alt-mysql` profile, so I'll fix that tomorrow. This change looks good, though.

**Comment by rhauch** [comment]
@christian-posta: I've refactored the profiles to allow the `-Palt-mysql` profile to be used with any command. See #14 and [DBZ-14](https://issues.jboss.org/browse/DBZ-14) for details.

**Comment by christian-posta** [comment]
Ah, very nice @rhauch thanks! I'll take a look!

**Pull request merged**